GitOps 1.9.2 release notes documentation. (#64185)
openshift/openshift-docs: modules/gitops-release-notes-1-9-2.adoc

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

:_content-type: REFERENCE

[id="gitops-release-notes-1-9-2_{context}"]
= Release notes for {gitops-title} 1.9.2

{gitops-title} 1.9.2 is now available on {product-title} 4.12 and 4.13.

[id="fixed-issues-1-9-2_{context}"]
== Fixed issues

> The following issues have been resolved in the current release:

Reviewer: ```suggestion
The following issue has been resolved in the current release:
```

Author: Done!

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

:_content-type: REFERENCE

[id="gitops-release-notes-1-9-2_{context}"]
= Release notes for {gitops-title} 1.9.2

{gitops-title} 1.9.2 is now available on {product-title} 4.12 and 4.13.

[id="errata-updates-1-9-2_{context}"]
== Errata updates

> === RHSA-2023:5029 - {gitops-title} 1.9.2 security update advisory

Reviewer: ```suggestion
[id="errata-security-update-advisory-1-9-2_{context}"]
=== RHSA-2023:5029 - {gitops-title} 1.9.2 security update advisory
```

Reviewer: @masauso-lungu Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

- All titles and section headings must have an anchor ID. The anchor ID must be similar to the title or section heading.
- You must add the {context} variable to the end of each anchor ID in module files.

Author: what about what is in line 12?

Reviewer: @masauso-lungu Check the anchor id change now.

Issued: 2023-09-07

The list of security fixes that are included in this release is documented in the following advisory:

> * link:https://errata.engineering.redhat.com/advisory/120042[RHBA-2023:120042]

Reviewer: ```suggestion
* link:https://access.redhat.com/errata/120042[RHBA-2023:120042]
```

Reviewer: @masauso-lungu Please remember to notice the change in the RHSA number if any for this advisory once the errata state is changed to SHIPPED LIVE.
